tests/org.eclipse.emfcloud.modelserver.client.tests/src/org/eclipse/emfcloud/modelserver/client/v2/ModelServerClientV2Test.java

```diff
@@ -0,0 +1,721 @@
+ ********************************************************************************/
+package org.eclipse.emfcloud.modelserver.client.v2;
+
+import static junit.framework.TestCase.assertTrue;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.fail;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.CancellationException;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+
+import org.eclipse.emf.common.command.BasicCommandStack;
+import org.eclipse.emf.common.command.Command;
+import org.eclipse.emf.common.util.URI;
+import org.eclipse.emf.ecore.EClass;
+import org.eclipse.emf.ecore.EObject;
+import org.eclipse.emf.ecore.EcoreFactory;
+import org.eclipse.emf.ecore.EcorePackage;
+import org.eclipse.emf.ecore.InternalEObject;
+import org.eclipse.emf.ecore.util.EcoreUtil;
+import org.eclipse.emf.edit.command.AddCommand;
+import org.eclipse.emf.edit.domain.AdapterFactoryEditingDomain;
+import org.eclipse.emf.edit.provider.ComposedAdapterFactory;
+import org.eclipse.emfcloud.modelserver.client.Model;
+import org.eclipse.emfcloud.modelserver.client.Response;
+import org.eclipse.emfcloud.modelserver.client.ServerConfiguration;
+import org.eclipse.emfcloud.modelserver.command.CCommand;
+import org.eclipse.emfcloud.modelserver.command.CCommandFactory;
+import org.eclipse.emfcloud.modelserver.common.ModelServerPathParametersV2;
+import org.eclipse.emfcloud.modelserver.common.ModelServerPathsV2;
+import org.eclipse.emfcloud.modelserver.common.codecs.DefaultJsonCodec;
+import org.eclipse.emfcloud.modelserver.common.codecs.EncodingException;
+import org.eclipse.emfcloud.modelserver.common.codecs.XmiCodec;
+import org.eclipse.emfcloud.modelserver.edit.EMFCommandType;
+import org.eclipse.emfcloud.modelserver.edit.command.AddCommandContribution;
+import org.eclipse.emfcloud.modelserver.emf.common.JsonResponse;
+import org.eclipse.emfcloud.modelserver.emf.common.JsonResponseMember;
+import org.eclipse.emfcloud.modelserver.emf.common.JsonResponseType;
+import org.eclipse.emfcloud.modelserver.emf.common.codecs.JsonCodec;
+import org.eclipse.emfcloud.modelserver.emf.common.codecs.JsonCodecV2;
+import org.eclipse.emfcloud.modelserver.jsonschema.Json;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.google.common.base.Charsets;
+
+import okhttp3.HttpUrl;
+import okhttp3.HttpUrl.Builder;
+import okhttp3.OkHttpClient;
+import okhttp3.mock.MockInterceptor;
+import okhttp3.mock.Rule;
+import okio.Buffer;
+
+public class ModelServerClientV2Test {
```

Comment:
All relevant classes have been forked when we started splitting the API. However, since we didn't actually provide a V2 Client for Java, the V2 Client and the V2 Tests are completely identical to V1 (Except for file names)